src/Uno.UWP/Storage/PathIO.cs

```diff
@@ -0,0 +1,298 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.Storage.Streams;
+using UwpUnicodeEncoding = Windows.Storage.Streams.UnicodeEncoding;
+
+namespace Windows.Storage
+{
+	/// <summary>
+	/// Provides helper methods for reading and writing a file using
+	/// the absolute path or Uniform Resource Identifier (URI) of the file.
+	/// </summary>
+	public static partial class PathIO
+	{
+		/// <summary>
+		/// Reads the contents of the file at the specified path or Uniform Resource Identifier (URI) and returns text.
+		/// </summary>
+		/// <param name="absolutePath">The path of the file to read.</param>
+		/// <returns>When this method completes successfully, it returns the contents of the file as a text string.</returns>
+		public static IAsyncOperation<string> ReadTextAsync(string absolutePath) =>
+			ReadTextTaskAsync(absolutePath).AsAsyncOperation();
```

Comment:
You should prefer to use `AsyncOperation.FromTask` which gives you access to the `CancellationToken`